test/functionalTest/harnessFunctions.sh

```diff
@@ -87,6 +87,50 @@ function dMsg()
 }
 
 
+# ------------------------------------------------------------------------------
+#
+# dbInitTenant - 
+#
+function dbInitTenant()
+{
+  role=$1
+
+  
+  host="${CB_DATABASE_HOST}"
+  if [ "$host" == "" ]
+  then
+    host="localhost"
+  fi
+  
+  port="${CB_DATABASE_PORT}"
+  if [ "$port" == "" ]
+  then
+    port="27017"
+  fi
+  
+  db=$host:$port/$1 
+
+  dMsg initializing database;
+
+  if [ "$role" == "CB t1" ]
```

Comment:
This looks funny ...
If the function is called with two params (CB t1, for example),  then  $1 will be "CB" and $2 will be "t1".
As role is set to $1 and after that not modified, the only way to get here is to call the function like this:

    dbInitTenant "CB t1"     OR
    dbInitTenant CB\ t1

Have a look at this, please :-)